ui/api/logging.go

```diff
@@ -1037,39 +1316,57 @@ func (lm *loggingMiddleware) GetRemoteTerminal(id, token string) (res []byte, er
 // ProcessTerminalCommand adds logging middleware to async function ProcessTerminalCommand.
 func (lm *loggingMiddleware) ProcessTerminalCommand(ctx context.Context, id, token, command string, res chan string) (err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method remote_terminal took %s to complete", time.Since(begin))
+		args := []interface{}{
+			slog.Any("duration", time.Since(begin)),
+			slog.String("thing_id", id),
+			slog.String("command", command),
+		}
 		if err != nil {
-			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			args = append(args, slog.String("err", err.Error()))
+			lm.logger.Error("Process terminal command failed to complete successfully", args...)
 		}
-		lm.logger.Info(fmt.Sprintf("%s without errors.", message))
+		lm.logger.Info("Process terminal command completed successfully", args...)
 	}(time.Now())
 
 	return lm.svc.ProcessTerminalCommand(ctx, id, token, command, res)
 }
 
 // GetEntities adds logging middleware to get entities method.
-func (lm *loggingMiddleware) GetEntities(token, item, name, domainID, permission string, page, limit uint64) (b []byte, err error) {
+func (lm *loggingMiddleware) GetEntities(token, entityType, entityName, domainID, permission string, page, limit uint64) (b []byte, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method get_entities took %s to complete", time.Since(begin))
+		args := []interface{}{
+			slog.Any("duration", time.Since(begin)),
+			slog.String("entity", entityType),
+			slog.String("entity_name", entityName),
+			slog.String("domain_id", domainID),
+			slog.String("permission", permission),
+			slog.Uint64("page", page),
+			slog.Uint64("limit", limit),
+		}
 		if err != nil {
-			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			args = append(args, slog.String("err", err.Error()))
+			lm.logger.Error("Get entities failed to complete successfully", args...)
 			return
 		}
-		lm.logger.Info(fmt.Sprintf("%s without errors.", message))
+		lm.logger.Info("Get entities completed successfully", args...)
 	}(time.Now())
 
-	return lm.svc.GetEntities(token, item, name, domainID, permission, page, limit)
+	return lm.svc.GetEntities(token, entityType, entityName, domainID, permission, page, limit)
 }
 
 // ErrorPage adds logging middleware to error page method.
 func (lm *loggingMiddleware) ErrorPage(errMsg string) (b []byte, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method ErrorPage took %s to complete", time.Since(begin))
+		args := []interface{}{
+			slog.Any("duration", time.Since(begin)),
+			slog.String("error_message", errMsg),
```

Comment:
```suggestion
			slog.String("error", errMsg),
```

I think `error` is just fine